Reinstate Token CA

hdpe opened the pull request:
This reverts a change made in 383fa7f where the Token CA was removed
from the trusted certs file, causing SSL handshake failure when TokenOS
attempts to contact the sample bank.

yuhanniu commented:
Hi @hdpe , thanks for the PR. I have created https://github.com/tokenio/bank-sample-java/pull/26 to add back the two intermediate CAs with comments.

The certificate you are adding here is the intermediate CA we use for production environment, so it shouldn't have fixed any connection issue for you in sandbox. Could you double check?

hdpe commented:
It's apparently also the intermediate cert for the sandbox environment.

As you can see by my screenshot, the CA for both prod and sandbox has CN "Token".

My PR reintroduces the cert with CN "Token" to the trusted-certs file. Without this, there is no cert with CN "Token" in the trusted-certs file, so any cert with CA "Token" will not be trusted!

![certs](https://user-images.githubusercontent.com/1238667/55147118-b4d19600-513d-11e9-9083-432d8d09ff8b.png)

hdpe commented:
I have verified that the cert I added to trusted-certs in 2d5342 is indeed the same as the one that I've added to my trusted-certs to allow Sandbox to talk to our bank.

This is the same as the one you've labelled "Token Intermediate CA Production" in your PR in e15d0c.
